app/views/publishers/vacancies/job_applications/tag.html.slim

```diff
@@ -11,20 +11,20 @@
 
     p.govuk-body
       ul
-        - @job_applications.each do |ja|
+        - @form.job_applications.each do |ja|
           li = ja.name
 
-= form_with(url: update_tag_organisation_job_job_applications_path(vacancy.id), scope: :publishers_job_application_status_form) do |f|
-  =  f.hidden_field :origin, value: @origin
-  - @job_applications.each do |ja|
+= form_with(model: @form, url: update_tag_organisation_job_job_applications_path(vacancy.id)) do |f|
+  = f.govuk_error_summary
+  = f.hidden_field :origin, value: @form.origin
+  - @form.job_applications.each do |ja|
     = f.hidden_field :job_applications, multiple: true, value: ja.id
 
-  = f.govuk_radio_buttons_fieldset :status do
-    = f.govuk_radio_button :status, :submitted
-    = f.govuk_radio_button :status, :unsuccessful
-    = f.govuk_radio_button :status, :reviewed
-    = f.govuk_radio_button :status, :shortlisted
+  = f.govuk_radio_buttons_fieldset :status, legend: nil do
+    - @form.class::VALID_STATUSES[..-2].each do |status|
```

Comment:
[Minor] I think `VALID_STATUSES.excluding("interviewing")` might be a bit easier to read/understand